Source/WebKit/UIProcess/API/mac/WKView.mm

```diff
@@ -88,86 +84,72 @@ @implementation WKView
 
 - (id)initWithFrame:(NSRect)frame processGroup:(WKProcessGroup *)processGroup browsingContextGroup:(WKBrowsingContextGroup *)browsingContextGroup
 {
-    return [self initWithFrame:frame contextRef:processGroup._contextRef pageGroupRef:browsingContextGroup._pageGroupRef relatedToPage:nil];
+    return nil;
 }
 
 - (id)initWithFrame:(NSRect)frame processGroup:(WKProcessGroup *)processGroup browsingContextGroup:(WKBrowsingContextGroup *)browsingContextGroup relatedToView:(WKView *)relatedView
 {
-    return [self initWithFrame:frame contextRef:processGroup._contextRef pageGroupRef:browsingContextGroup._pageGroupRef relatedToPage:relatedView ? relatedView.pageRef : nil];
+    return nil;
 }
 
 - (void)dealloc
```

Comment:
Should just be able to remove the whole method?

Reply:
(it probably is true for dealloc, but I'm going to leave it with the rest for now)

Reply:
I think it's clearly true for any that just call super :) all bets are off for the others. But, yes, this is fine too.